packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -131,13 +139,12 @@ const TransferPanel = (): JSX.Element => {
     }
     setTransferring(true)
     try {
-      const amount = isDepositMode ? l1Amount : l2Amount
+      let amount = isDepositMode ? l1Amount : l2Amount
+      if (amount === '' && isBridgingANewStandardToken) {
```

Comment:
Do we ever have `amount === ''`? I don't think so

Reply:
the input validation only lets you trigger the `deposit/deploy` modal if you have an input.
you need to explicitly set the amount as 0 so not sure why `amount === ''`